chef/cookbooks/crowbar/templates/default/crowbar-chef-upgraded.sh.erb

```diff
@@ -45,5 +45,19 @@ rm -f /usr/sbin/crowbar_join_tmp
 systemctl start cron
 systemctl enable cron
 
+# Remove nova-cert service which has been deprecated in Newton and is not present in Pike
+<% if @nova_controller %>
+
```

Comment:
This of course means we're now running this on every single nova-controller, while it would be enough to just run it once. But I guess we can live with that.

Reply:
Yes, I know, I wanted to be sure :-)
And if no service is found, no deleting will happen